.github/workflows/test-docker-image-creation.yml

```diff
@@ -0,0 +1,27 @@
+name: Test multi-arch Docker Image creation
```

Comment:
can't we use GHA's conditions and consolidate both YAML files?

Reply:
Fair point - have done so.